File: contest1/pB.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ios_base::sync_with_stdio(0);
    cin.tie(0);
    // freopen("input.txt", "r", stdin);
    /// bibliografia: guide to competitive programming
    /*
        revisar:
        getline() ///hay distintos parametros aqui.

    */
    vector<int> nums;
    int n;
    cin >> n;
    for (int i = 0; i < n; i++) {
        int x;
        cin >> x;
        nums.push_back(x);
    }
    sort(nums.begin(), nums.end(), greater<int>());
    int price = 0;
    for (int i = 0; i < n; i++) {
        if (i % 3 == 2) {
            continue;
        }
        price += nums[i];
    }
    cout << price << endl;
    return 0;
}

File: lab1/pB.py
n = int(input())
nums = input().split(" ")
nums = [int(x) for x in nums]
nums.sort()

for i in range(len(nums)):
    if (i + 1) != nums[i]:
        print(i + 1)
        break
if nums[-1] != len(nums) + 1:
    print(len(nums) + 1)
